SimPEG/Mesh/MeshIO.py

```diff
@@ -0,0 +1,476 @@
+import numpy as np, os
```

Comment:
I don't think this should be here?? We should now be using the MeshIO within discretize